src/QMCHamiltonians/PairCorrEstimator.cpp

```diff
@@ -164,7 +164,7 @@ void PairCorrEstimator::registerCollectables(std::vector<ObservableHelper>& h5li
   int offset = my_index_;
   for (int i = 0; i < gof_r_prefix.size(); ++i)
   {
-    h5list.emplace_back(gof_r_prefix[i]);
+    h5list.push_back({{name_, gof_r_prefix[i]}});
```

Comment:
Suspicious change. Why two element now?

Reply:
I started to change this so it wouldn't just dump its variables in the the global group.  But we shouldn't change it in this PR.